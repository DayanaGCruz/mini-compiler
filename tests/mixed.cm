float scaleAndShift(int base, float scale) {
  float result;
  result = base * scale;
  return result;
}

void main() {
  int numbers[2];
  float computed;
  struct Pair {
    int first;
    char mark;
  };
  struct Pair p;

  numbers[0] = 3;
  numbers[1] = 4;
  computed = scaleAndShift(numbers[0] + numbers[1], 1.5);
  print(computed);

  p.first = numbers[0] * numbers[1];
  p.mark = 'B';
  print(p.first);
  print(p.mark);
}
