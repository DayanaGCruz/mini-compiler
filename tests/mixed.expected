10.50000000
12
B
